src/crypto/pow_hash/cn_slow_hash_soft.cpp

```diff
@@ -187,8 +182,8 @@ struct aesdata
 
 inline uint32_t sub_word(uint32_t key)
 {
-	return (saes_sbox[key >> 24] << 24) | (saes_sbox[(key >> 16) & 0xff] << 16) |
-		   (saes_sbox[(key >> 8) & 0xff] << 8) | saes_sbox[key & 0xff];
+	return (saes_sbox[key >> 24 ] << 24)   | (saes_sbox[(key >> 16) & 0xff] << 16 ) | 

```

Comment:
why do you add wild spaces. due to the tab indention it is not possible to align code via spaces.

Reply:
I will run it through clang format